dataland-frontend/src/components/resources/dataTable/conversion/DataPoints.ts

```diff
@@ -139,3 +133,37 @@ function doesAnyDataPointPropertyExist(dataPointProperties: DatapointProperties
       (dataPointProperties?.dataSource && dataPointProperties?.dataSource.fileReference.trim().length > 0))
   );
 }
+
+/**
+ * Builds the displayValue object for the DataPointWrapperDisplayComponent when quality, comment and data source are NOT
+ * all empty.
+ * @param inputValue the original value to wrap
+ * @param fieldLabel the label of the field to wrap
+ * @param datapointProperties the properties of the datapoint to wrap
+ * @returns the built displayValue object
+ */
+function buildDisplayValueWhenDataPointMetaInfoIsAvailable(
+  inputValue: AvailableMLDTDisplayObjectTypes,
+  fieldLabel: string,
+  datapointProperties: DatapointProperties | undefined | null
+): MLDTDisplayComponentTypes[MLDTDisplayComponentName.DataPointWrapperDisplayComponent] {
+  let innerContent: AvailableMLDTDisplayObjectTypes;
+  const isOnlyQualityProvided =
+    datapointProperties?.dataSource == undefined &&
+    datapointProperties?.comment == undefined &&
+    inputValue.displayValue == '';
+
+  if (isOnlyQualityProvided) {
+    innerContent = formatStringForDatatable(datapointProperties?.quality);
+  } else {
+    innerContent = inputValue.displayValue == '' ? formatStringForDatatable(ONLY_AUXILIARY_DATA_PROVIDED) : inputValue;
+  }
+
+  return {
+    innerContents: innerContent,
+    quality: isOnlyQualityProvided ? undefined : datapointProperties?.quality ?? undefined,
```

Comment:
I don't understand. If quality is provided (`isOnlyQualityProvided == true`) you set quality to `undefined`? And if it is false, i.e. if eg. displayValue is not empty, you set it to `datapointProperties.quality` unless it is null? This means you set quality to undefined even if datapointProperties.quality exists and isOnlyQualityProvided is true, right? Why?

Reply:
Ich trickse die Anzeigekomponente aus:

Wenn NUR quality da ist, dann möchte ich, dass die Anzeigekomponente das wie einen "Wert" anzeigt.
Daher verpacke ich in diesem Fall den String, der in `dataProperties.quality` steckt in den `innerContents`-Slot, und nehme ihn aus dem `quality` slot raus.

=> Das führt dazu, dass der Quality-String literally im Frontend als "Wert" auftraucht.


Hatte vorher einige andere Wege ausprobiert, das zu erreichen, die aber in Katastrophen geendet sind.
Grund: Der Anzeige-Code von Datapoints ist unfassbar verschachtelt und kleinste Änderungen sprengen Dinge an x Stellen. Daher habe ich den belassen wie er ist, und stattdessen an dieser Stelle hier direkt an der "Quelle" meine Änderungen vorgenommen.

Reply:
Wie besprochen: Habe die description dieser Funktion jetzt so angepasst, dass man dieses "Austricksen" hier als Code-Leser versteht.